src/sidebar/components/annotation-action-bar.js

```diff
@@ -101,6 +118,7 @@ function AnnotationActionBar({
           isPressed={true}
           icon="flag--active"
           title="Annotation has been reported to the moderators"
+          onClick={()=>{}}
```

Comment:
Funny, we just talked about this. I guess this may be the one and only place we don't have an `onClick`, but it's also something that needs to be fixed. We're going to eventually have to add support for un-flagged an annotation. For now I just added a no-op which seems reasonable. I hesitate to make `onClick` not required. Thoughts?

Reply:
I agree, passing a no-op callback here seems reasonable. In almost all cases having a `Button` without an `onClick` would be a mistake.